packages/wonder-stuff-ci/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "publishConfig": {
+        "access": "public"
+    },
+    "engines": {
+        "node": ">=16"
+    },
+    "name": "@khanacademy/wonder-stuff-ci",
+    "version": "0.0.1",
+    "description": "Functions for creating automation and scripts.",
+    "scripts": {
+        "test": "bash -c 'yarn --silent --cwd \"../..\" test ${@:0} $($([[ ${@: -1} = -* ]] || [[ ${@: -1} = bash ]]) && echo $PWD)'"
+    },
+    "devDependencies": {
+        "@types/node": "^18.15.11",
+        "ws-dev-build-settings": "^1.1.0"
```

Comment:
What is the ws-dev-build-settings? Should I be adding the @types/node into this?

Reply:
This is internal to wonder-stuff itself, these are the shared tools that we use to help with test running, builds, etc. I'm not sure if it makes sense to have the node types, or not! I'd say "no" only because not everything in wonder-stuff is designed to run in Node.